pages/mealcenter.js

```diff
@@ -0,0 +1,52 @@
+import styled from "styled-components";
+import FoodCard from "../components/FoodCard";
+import recipes from "../flattened-download.json";
+
+export default function Mealcenter({ favourites }) {
+  const favouriteRecipes = recipes.filter((recipe) => favourites.includes(recipe.id));
+
+  return (
+    <main>
+      <h1>WTF? What to food</h1>
+      {favouriteRecipes.length === 0 ? (
+        <p>Nothing here..</p>
+      ) : (
+        <>
+          <h2>my super tasty selection of recipes:</h2>
+          <ul>
+            {favouriteRecipes.map((recipe) => (
+              <FoodCard recipe={recipe} key={recipe.id} />
+            ))}
+          </ul>
+        </>
+      )}
+
+      <SytledNav>
+        <StyledImgBox href="/">
+          <img src="/icons/iconHome.png" />
+        </StyledImgBox>
+        <StyledImgBox href="/mealcenter">
+          <img src="/icons/iconMealCenter.png" />
+        </StyledImgBox>
+      </SytledNav>
+    </main>
```

Comment:
See my previous comment regarding your navigation. As the navigation will be displayed on every page in your app, I strongly suggest to create a component for it.